profile/software/members/eddie.md

```diff
@@ -0,0 +1,36 @@
+# 에디 (Eddie)
+
+TODO: Add image here.
+
+## Position
+
+Python Application Developer
+
+## About me
+
+#### Bio:
+
+Edward L. Forgacs is a technology professional with a robust background in creating innovative solutions for Fortune 500 companies like Bank of America, Google, HP Inc., and IBM. He has served in both tech leadership and individual contributor capacities. He possesses an innate ability to merge cutting-edge AI solutions with strategic business objectives. He excels in leveraging programming languages like Python, Java, and C++ to develop tools that optimize software performance, improve search algorithms, and enhance natural language processing systems. His expertise spans across web frameworks, project management tools, and automation, making him a versatile and impactful contributor in the technology domain. With a Master of Science in Computer Science from New York University and a Bachelor of Arts in International Affairs from The George Washington University, Edward brings forth a unique blend of technical prowess and global perspective.
+
+#### Fun facts about me:
+
+- I earned my private pilot license when I was 17 years old.
+- I like to play musical instruments, and in my free time I play piano and some guitar.
```

Comment:
![image](https://github.com/PLAIF-dev/.github/assets/47849054/75dd2af2-da5c-480c-87dd-863e77e0d481)
Oh~ Woochul, who sits next to me, can play piano and guitar too.
[Here](https://www.youtube.com/@painogita6500) is his YouTube account.

Reply:
Moreover, his account name is "painogita"!!

Reply:
Very nice! Woochul and I should have a jam session on piano/guitar together sometime, maybe we can start a band! Haha

Reply:
> Very nice! Woochul and I should have a jam session on piano/guitar together sometime, maybe we can start a band! Haha

Unfortunately, he cannot read this comment. :cry: 
He is not a software developer.

Reply:
> > Very nice! Woochul and I should have a jam session on piano/guitar together sometime, maybe we can start a band! Haha
> 
> Unfortunately, he cannot read this comment. 😢 He is not a software developer.

Awww, okay. But the sentiment is still there, haha! One day I'm sure we'll play some music together, it's inevitable XD